uaclient/contract.py

```diff
@@ -280,18 +282,11 @@ def request_updated_contract(
                 status.MESSAGE_CONTRACT_EXPIRED_ERROR
             )
     user_errors = []
-    for name, entitlement in sorted(cfg.entitlements.items()):
-        if entitlement["entitlement"].get("entitled"):
-            # Obtain each entitlement's accessContext for this machine
-            new_access = contract_client.request_resource_machine_access(
```

Comment:
Have we confirmed with the contract server team that they are prepared for any consequences of us not hitting this endpoint any longer?

Reply:
We had in IRC with mhilton and cmars before starting this work. And their consensus was that the can (and should be) determinig KPI attach metrics from their backend service endpoints (like livepatch of various PPA paths during apt-update calls).  We may have to restate that we have completed this effort again to confirm that they aren't relying on information from ua client attaching via those routes.

Reply:
pinged in #ua IRC again about this cmars will chat with the team tomorrow.

Reply:
confirmed in IRC today from alnvdl and cmars that this is not an issue.